server/src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -28,10 +25,6 @@ public class CommonTermsQueryBuilder extends AbstractQueryBuilder<CommonTermsQue
     public static final String COMMON_TERMS_QUERY_DEPRECATION_MSG = "Common Terms Query usage is not supported. "
         + "Use [match] query which can efficiently skip blocks of documents if the total number of hits is not tracked.";
 
-    @UpdateForV9(owner = UpdateForV9.Owner.SEARCH_RELEVANCE) // v7 REST API no longer exists: eliminate ref to RestApiVersion.V_7
-    public static ParseField NAME_V7 = new ParseField("common").withAllDeprecated(COMMON_TERMS_QUERY_DEPRECATION_MSG)
```

Comment:
Looks like there is more to do around this cleanup. Can you do some digging to figure it out? I think we may be able to remove the query builder entirely.

Reply:
That's right, it was added in this pr https://github.com/elastic/elasticsearch/pull/75896